src/components/TranslationLeaderboard.js

```diff
@@ -0,0 +1,245 @@
+
+  const showLess = () => {
+    updateFilterAmount(10)
+  }
+
+  const showMore = () => {
+    updateFilterAmount(50)
+  }
+
+  return (
+    <Content>
+      <ButtonRow>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("monthData")}
+          isSelected={dateRangeType === "monthData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-month-view" />
+        </ButtonSecondaryStyled>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("quarterData")}
+          isSelected={dateRangeType === "quarterData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-quarter-view" />
+        </ButtonSecondaryStyled>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("allTimeData")}
+          isSelected={dateRangeType === "allTimeData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-all-time-view" />
+        </ButtonSecondaryStyled>
+      </ButtonRow>
+      <Table>
+        <Header>
+          <ItemNumber>#</ItemNumber>
+          <TextContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-translator" />
+          </TextContainer>
+          <LangaugeContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-language" />
+          </LangaugeContainer>
+          <WordsContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-total-words" />
+          </WordsContainer>
+        </Header>
+        {leaderboardData[dateRangeType].data
+          .filter((item, idx) => idx < filterAmount)
+          .map((item, idx) => {
+            const { user, translated, languages } = item
+            let emoji = null
+            if (idx === 0) {
+              emoji = ":trophy:"
+            } else if (idx === 1) {
+              emoji = ":2nd_place_medal:"
+            } else if (idx === 2) {
+              emoji = ":3rd_place_medal:"
+            }
+            return (
+              <Item key={idx}>
+                {emoji ? (
+                  <Emoji mr={"1rem"} text={emoji} />
+                ) : (
+                  <ItemNumber>{idx + 1}</ItemNumber>
+                )}
+                <TextContainer>
+                  <Avatar src={user.avatarUrl} />
+                  <div>
+                    {user.username}
+                    <LanguageMobile>{languages[0]?.name || ""}</LanguageMobile>
+                  </div>
+                </TextContainer>
+                <LangaugeContainer>
+                  {languages[0]?.name || ""}
+                </LangaugeContainer>
+                <WordsContainer>
+                  <StyledEmoji
+                    ml={"1rem"}
+                    mr={"1rem"}
+                    size={1.5}
```

Comment:
The sizing here doesn't appear to be enforced - I'm seeing a range of sizes that looks to vary depending on the size of the `translated` number:
![Image 2021-11-29 at 7 47 00 PM](https://user-images.githubusercontent.com/8097623/143971050-52cce4e0-ae6a-454a-8999-0099f8097c66.jpg)